scripts/travis/podman-test.sh

```diff
@@ -13,6 +18,11 @@ apt-get update -qq
 
 apt-get install -qqy podman containernetworking-plugins
 
+# overlaysfs behaves differently on Ubuntu and breaks CRIU
+# https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1857257
+#sed -e "s,driver.*$,driver = \"vfs\",g" -i /etc/containers/storage.conf
+echo 'driver = "vfs"' > /etc/containers/storage.conf
```

Comment:
```sh
sed -i 's/^\(driver =\).*/\1 "vfs"/' /etc/containers/storage.conf
```

Reply:
Thanks. I am still trying to figure out how to configure it all correctly. Lot's of changes all the time.